src/System.Drawing.Common/tests/IconTests.cs

```diff
@@ -495,17 +492,92 @@ public void ToBitmap_BitmapIcon_Success(Icon icon)
         }
 
         [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
-        public void ToBitmap_PngIcon_Success()
+        public void ToBitmap_BitmapIconFromHandle_ReturnsExpected()
+        {
+            // Handle referring to icon without any colour.
+            using (var originalIcon = new Icon(Helpers.GetTestBitmapPath("48x48_one_entry_1bit.ico")))
+            using (Icon icon = Icon.FromHandle(originalIcon.Handle))
+            {
+                ToBitmap_BitmapIcon_ReturnsExpected(icon);
+            }
```

Comment:
Why did this one need to be separated out from ToBitmap_TestData?  It'd be worth a comment.